app/controllers/sessions_controller.rb

```diff
@@ -11,7 +12,7 @@ def create
     @user = find_user
 
     if @user && (@user.mfa_enabled? || @user.webauthn_credentials.any?)
-      setup_webauthn_authentication
+      setup_webauthn_authentication(session_options: { "user" => @user.id }, form_url: webauthn_create_session_path)
```

Comment:
I suggest flipping these arguments so they're in the same order as the method signature.